core/types/transaction.go

```diff
@@ -295,6 +295,13 @@ func (tx *Transaction) RawSignatureValues() (*big.Int, *big.Int, *big.Int) {
 // Transactions is a Transaction slice type for basic sorting.
 type Transactions []*Transaction
 
+// RecentTxsStats is a recent transactions stats map tracking stats like BlockTxsCounts.
+type RecentTxsStats map[uint64]BlockTxsCounts
```

Comment:
put these temp code into other places like node.go. We should keep Transaction type code clean.